lib/api/v3/work_packages/schema/work_package_schema_representer.rb

```diff
@@ -41,7 +41,7 @@ class WorkPackageSchemaRepresenter < ::API::Decorators::SchemaRepresenter
           cached_representer key_parts: %i[project type],
                              dependencies: -> {
                                Authorization.roles(User.current, represented.project).map(&:permissions).sort +
-                                 [Setting.work_package_done_ratio]
+                                 [Setting.work_package_done_ratio, I18n.locale]
```

Comment:
the locale should be part of any cachable representer: https://github.com/opf/openproject/blob/dev/lib/api/caching/cached_representer.rb#L80

The question then is, why was that not factored in.